.wwhrd.yml

```diff
@@ -0,0 +1,15 @@
+---
+blacklist:
+  # https://www.apache.org/licenses/GPL-compatibility.html
+  - GPL-2.0
+
+whitelist:
+  - Apache-2.0
+  - MIT
+  - NewBSD
+  - FreeBSD
+  - ISC
+
+exceptions:
+  - github.com/davecgh/go-spew/spew/...
```

Comment:
why are those two libs in expectations?

Reply:
The licenses for these pkg s are not detected by https://github.com/ryanuber/go-license which is vendored by github.com/frapposelli/wwhrd(which is used by this PR)

Debugging the reason :) . Will post a pr on their repo, if I find a fix

Reply:
If we are going this lets do it right. There needs to be comment explaining what is a reason for each exception.

Reply:
Ack I'll add the comment and try to send a fix pr to https://github.com/ryanuber/go-license for detection of isc licenses at the earliest.. but can we get this PR in with a tracker issue on our side for ISC fix?